### ISSUE TITLE
sp.disable_function.function and value_r not working as expected

### ISSUE THREAD

**Issue by JQuags** [nbs-system/snuffleupagus#181]
PHP version 7.1
Latest git combined.

I'm trying to determine if I am mis-understanding this or it is not working.

There are these default rules included:

```python
sp.disable_function.function("require_once").value_r("\.php$").allow();
sp.disable_function.function("include_once").value_r("\.php$").allow();
sp.disable_function.function("require").value_r("\.php$").allow();
sp.disable_function.function("include").value_r("\.php$").allow();
sp.disable_function.function("require_once").drop()
sp.disable_function.function("include_once").drop()
sp.disable_function.function("require").drop()
sp.disable_function.function("include").drop()
```

I expected include functions to be denied that did not end in `.php$`. To test I attempted to include a file called test.ico which was allowed. I also tried the reverse just to block `.ico` (with similation)

```python
sp.disable_function.function("require_once").value_r("\.ico$").drop().simulation();
sp.disable_function.function("include_once").value_r("\.ico$").drop().simulation();
sp.disable_function.function("require").value_r("\.ico$").drop().simulation();
sp.disable_function.function("include").value_r("\.ico$").drop().simulation();
```

Doing the above blocks all include calls, not just those ending with `.ico`.

**Comment by kkadosh** [comment]
I managed to reproduce the bug. I will investigate further.

**Comment by jvoisin** [comment]
Thank you for trying Snuffleupagus ♥ We managed to reproduce your issue on our end.

A possible workaround would be something like this:

```python
sp.disable_function.function("include_once").param("").value_r("\.ico$").drop(); ?
```

We're working on making the `param("")` filter default on language constructs.

**Comment by jvoisin** [comment]
It should be fixed by ca3be84076521c4bb053511775c94c0b195aeac8

**Comment by JQuags** [comment]
Thanks everyone. Will further test and looking forward to helping out this project where I can.
